torchgeo/datasets/ucmerced.py

```diff
@@ -143,6 +144,20 @@ def __init__(
             is_valid_file=is_in_split,
         )
 
+    def _load_image(self, index: int) -> Tuple[Tensor, Tensor]:
+        """Load a single image and it's class label.
+
+        Args:
+            index: index to return
+
+        Returns:
+            the image
+            the image class label
```

Comment:
```suggestion
            the image and class label
```
Python functions only return one thing, in this case a tuple.